index.html

```diff
@@ -786,8 +786,8 @@ <h3>Multiple Schemas</h3>
             }
             </pre>
           </p>
-          <p class="issue" data-number="136">
-            Add warning for improperly formed schemas and risks associated with using multiple schemas.
+          <p>
+            Composition using `allOf` into a JSON Schema can easily result in a schema for which all JSON documents will fail to validate. Such a situation may happen when multiple schemas reference the same property. Schema authors are advised to ensure that there exists at least one JSON document for which schema validation would pass.
```

Comment:
this looks good - can add a sentence such as ...

"Implementers are advised to test their schemas against a set of sample input documents before introducing any real world usage"?

Reply:
Added that and some more.

Reply:
thanks!